docs/docs/reference-react.md

```diff
@@ -1,24 +1,67 @@
 ---
-id: reference-react
+id: react-api
 title: React Top-Level API
 layout: docs
 category: Reference
-permalink: docs/reference-react.html
+permalink: docs/react-api.html
 redirect_from: "/docs/reference.html"
 ---
 
 `React` is the entry point to the React library.
 
```

Comment:
Can we add a section on importing here? We had a few sentences which you removed due to my complaint but I want them to be there, just more descriptive.

"If you use React as a script tag, these top-level APIs are available on the `React` global. If you use ES6 with npm, you can write `import React from 'react'`. If you use ES5 with npm, you can write `var React = require('react')`."

Same for all other top level exports like ReactDOM and ReactDOMServer.